helm/oncall/templates/NOTES.txt

```diff
@@ -34,11 +34,6 @@
 
   🔗 Connect Grafana OnCall Plugin to Grafana OnCall backend:
 
-     Issue the one-time token to connect Grafana OnCall backend and Grafana OnCall plugin by running these commands:
-
-        export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "oncall.name" . }},app.kubernetes.io/instance={{ .Release.Name }},app.kubernetes.io/component=engine" -o jsonpath="{.items[0].metadata.name}")
-        kubectl exec -it $POD_NAME --namespace {{ .Release.Namespace }} -- bash -c "python manage.py issue_invite_for_the_frontend --override"
-
      Fill the Grafana OnCall Backend URL:
 
           http://{{ include "oncall.engine.fullname" . }}:8080
```

Comment:
TODO: remove the lines above as well

Reply:
you mean to remove the "Fill the Grafana URL:" section, correct? Because I think you would still need to configure the "Grafana OnCall Backend URL" input as there would be no way to provide that in env var in the UI build for helm releases. Let's chat more about this on Monday.